.github/workflows/go.yml

```
@@ -61,8 +61,7 @@ jobs:
     - uses: golangci/golangci-lint-action@v4
       name: Install golangci-lint
       with:
-        version: latest
-        args: --version  # make lint will run the linter
+        version: latest  # make lint will run the linter
```

This will run the linter twice (`make lint` does it too).
We can keep the old hack in place but with a different argument:

```suggestion
        version: latest
        # Hack: Use the official action to download, but not run.
        # make lint below will handle actually running the linter.
        args: --help
```